src/core/Akka/Event/LogEvent.cs

```diff
@@ -62,7 +89,7 @@ protected LogEvent()
         /// <returns>A <see cref="System.String" /> that represents this LogEvent.</returns>
         public override string ToString()
         {
-            return string.Format("[{0}][{1}][Thread {2}][{3}] {4}", LogLevel().ToString().Replace("Level", "").ToUpperInvariant(), Timestamp, Thread.ManagedThreadId.ToString().PadLeft(4, '0'), LogSource, Message);
+            return string.Format("[{0}][{1}][Thread {2}][{3}] {4}", LogLevel().PrettyNameFor(), Timestamp, Thread.ManagedThreadId.ToString().PadLeft(4, '0'), LogSource, Message);
```

Comment:
Given 4 log levels, it will be much faster to make a switch statement on that enum, than trying to load ldf for dictionary field, then get enum hash, find the bucket inside the hash, then do enum equality comparison, and eventually load the string itself (hopefully still lying on the same cache line)

Reply:
That's a good call; we haven't changed the number of log levels ever so it's safe to do that. I've been in the midst of writing code like this for a brand new project where the enums are being developed regularly so that's where my head has been at.